dot/sync/chain_sync_integeration_test.go

```diff
@@ -480,7 +484,7 @@ func TestValidateBlockData(t *testing.T) {
 	err = cs.validateBlockData(req, &types.BlockData{
 		Header: &types.Header{},
 	}, "")
-	require.ErrorIs(t, err, errNilBodyInResponse)
+	require.Equal(t, errNilBodyInResponse, err)
```

Comment:
Keep it as ErrorIs in case we wrap it in the future

Reply:
Updated.